packages/cli/src/commands/transfer.ts

```diff
@@ -0,0 +1,63 @@
+import { ZWeb3 } from 'zos-lib';
+import transfer from '../scripts/transfer';
+import { promptIfNeeded, networksList, InquirerQuestions } from '../prompts/prompt';
+import { isValidUnit } from '../utils/units';
+import ConfigVariablesInitializer from '../models/initializer/ConfigVariablesInitializer';
+
+const name: string = 'transfer';
+const signature: string = name;
+const description: string = 'send funds to a given address';
+
+const register: (program: any) => any = (program) => program
+  .command(signature, undefined, { noHelp: true })
+  .usage('[options]')
+  .description(description)
+  .option('--to <to>', 'specify recipient address')
+  .option('--value <value>', 'the amount of ether units to be transferred')
+  .option('--unit <unit>', 'unit name. If none is given, \'ether\' will be used. To see supported units, go to: https://web3js.readthedocs.io/en/1.0/web3-utils.html#id85')
```

Comment:
Let's avoid tying ourselves to web3 documentation. I'd just list here the most common ones (wei, kwei, gwei, milli, ether).